packages/field-base/src/input-mixin.js

```diff
@@ -141,7 +141,7 @@ export const InputMixin = dedupingMixin(
        *
        * @private
        */
-      __inputValuePopulatedChanged() {
+      _hasInputValueChanged() {
```

Comment:
Two more comments:

1. Should we rename the event to be `has-input-value-changed`?
2. With the current logic, the event would be also fired for default `false` value. Consider using something like this to check it the value was actually toggled by the user:

```js
_hasInputValueChanged(hasValue, oldHasValue) {
  if (hasValue || oldHasValue) {
    // fire an event ...
  }
}
```

Reply:
Have renamed it already 👌 
Updated. Now the event is not fired for the default value.